### ISSUE TITLE
"Send Later" column displays in Inbox upon startup of TB

### ISSUE THREAD

**Issue by dapimentel** [kewisch/quickmove-extension#47]
When I start TB (currently 78.3.1 (64-bit) on Red Hat Enterprise Linux Workstation release 7.9 (Maipo), The "Send Later" column displays in Inbox in spite of disabling it. See the following image:
> ![image](https://user-images.githubusercontent.com/16899342/100630212-7cf1fd00-32e7-11eb-8a2d-023a37d196f7.png)

When I navigate to another folder (i.e., Drafts) and back to the Inbox, then "Send Later" column disappears.

I am currently using QM version 1.9.0:
> ![image](https://user-images.githubusercontent.com/16899342/100630434-c7737980-32e7-11eb-8f39-68854e74508d.png)

There seems to be a preferences initialization problem upon startup.

**Comment by kewisch** [comment]
Is this specifically related to the add-on? Does it go away when the add-on is uninstalled? Have you tried clicking on the folder column picker to remove that column from the view?

**Comment by dapimentel** [comment]
@kewisch,
I assumed that this was specifically related to the add-on, since this add-on is the only one that that I have installed that has the "Send Later" column.
Unfortunately, I can no longer replicate either the problem or the environment in which I observed it. My corporate IT has moved on to using Thunderbird 78.12.0 (64-bit) and Red Hat Enterprise Linux release 8.4 (Ootpa), and I have not observed the problem recently.
